pyitau/main.py

```diff
@@ -87,6 +91,71 @@ def get_statements_from_month(self, month=1, year=2001):
         )
         return response.json()
 
+    def get_pix(self, days=90):
+        """Get and return the pix transaction for the last {days}."""
+
```

Comment:
**Suggestion:** Avoid using magic numbers. Instead, define them as constants at the top of your file or class for better readability and maintainability.
```suggestion
    DEFAULT_DAYS = 90
    def get_pix(self, days=DEFAULT_DAYS):
        """Get and return the pix transaction for the last {days}."""
```